lib/MultiModule.js

```diff
@@ -19,7 +20,9 @@ MultiModule.prototype = Object.create(Module.prototype);
 MultiModule.prototype.constructor = MultiModule;
 
 MultiModule.prototype.identifier = function() {
-	return "multi " + this.name;
+	return "multi " + this.dependencies.map(function(d) {
+		return path.resolve(this.context, d.request);
+	}.bind(this)).join("!");
```

Comment:
`readableIdentifier` should reflect `identifier` but in a human readable way. You can use `requestShortener.shorten` to generate a readable identifier from a request.

---

Other issue: identifier should only use `d.request` and not `this.context`. And don't join them by `!`. This doesn't make sense here (`!` is for loaders). Maybe just join them by `|` or space.